tower-sessions-core/src/session.rs

```diff
@@ -857,29 +856,48 @@ impl Session {
 /// Id::default();
 /// ```
 #[derive(Copy, Clone, Debug, Deserialize, Serialize, Eq, Hash, PartialEq)]
-pub struct Id(pub Uuid); // TODO: By this being public, it may be possible to override UUIDv4,
-                         // which is undesirable.
+pub struct Id(pub i128); // TODO: By this being public, it may be possible to override the
+                         // session ID, which is undesirable.
 
 impl Default for Id {
     fn default() -> Self {
-        Self(Uuid::new_v4())
+        use rand::prelude::*;
```

Comment:
Is there a downside to moving these imports to the top-level module? I think the service will end up effectively guaranteeing that these are brought in anyway because we use these impls there.

Reply:
Not really. It's admittedly an odd style I've picked up and didn't think about -- if I'm importing something only used in one function, I usually import it only where it's needed, to make it clearer where it's being used/where imports are coming from.

I can move those imports up to the top-level if you'd like. :smile:

Reply:
As I'm thinking about this, while currently the service is defined in core, ideally it wouldn't be that way in the future and I suppose in that case there might be some benefit to not bringing in e.g. `rand::prelude`.  I think it's fine either way.